pages.it/common/ansible-galaxy.md

```diff
@@ -22,3 +22,11 @@
 - Crea un nuovo ruolo:
 
 `ansible-galaxy init {{nome_nuovo_ruolo}}`
+
+- Acquisisci informazioni su un ruolo di un utente
+
+`ansible-galaxy role info {{username}}.{{nome_ruolo}}`
+
+- Acquisisci informazioni su una collection:
+
+`ansible-galaxy collection info {{username}}.{{collection_name}}`
```

Comment:
I don't speak Italian but I think this is the correct translation for the arguments:
```suggestion
`ansible-galaxy collection info {{nome_utente}}.{{nome_raccolta}}`
```